python/pyarrow/table.pxi

```diff
@@ -1016,6 +1064,28 @@ cdef class RecordBatch(_PandasConvertible):
             entries.append((name, column))
         return ordered_dict(entries)
 
+    def to_pylist(self, index=None):
+        """
+        Convert the RecordBatch to a list of dictionaries of rows.
+
+        Parameters
+        ----------
+        index: list
+            A list of column names to index.
```

Comment:
Is this `index` keyword needed? (it's to select a subset of columns to export) Eg `to_pydict` doesn't have it (we should probably add it there as well if we want to keep it)

We have nowadays the `select()` method, so it is relatively straightforward to do `table.select([...]-.to_pylist()` instead of `table.to_pylist(index=[...])`. 
Or if we keep it, I would call it something else as `index`, but for example rather `columns=[..]`.

Reply:
Thanks for all the info! Will remove it - I thought it would be good to have it before =)